services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from db.models import Ticket, Order
+
+
+def create_order(
+    tickets: list[dict], username: str, date: str = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        [
+            Ticket.objects.create(
+                movie_session_id=ticket["movie_session"],
+                row=ticket["row"],
+                seat=ticket["seat"],
+                order=order
+            )
+            for ticket in tickets
+        ]
```

Comment:
I personally don't like the usage of list comprehension here, but it's probably a question of preference.

Reply:
Thanks's for review)
Fixed.